src/System.Linq.Parallel/src/System/Linq/Parallel/Utils/Sorting.cs

```diff
@@ -65,14 +65,12 @@ internal class SortHelper<TInputOutput, TKey> : SortHelper<TInputOutput>, IDispo
             Contract.Assert(sharedkeys != null);
```

Comment:
@chcosta - Chris here is the change we looked at earlier that failed when building with Rosyln.